core/src/main/java/org/infinispan/protostream/config/AnnotationAttributeConfiguration.java

```diff
@@ -17,6 +17,11 @@ public interface AnnotationAttributeConfiguration {
     */
    String name();
 
+   /**
+    * The optional package name
+    */
+   String packageName();
```

Comment:
does not seem to be used anywhere. Neither `Builder packageName(String packageName);`

Reply:
Used by Infinispan (see other comment)